### ISSUE TITLE
Add a blog and allow contributions from the community?

### ISSUE THREAD

**Issue by paul-hammant** [lorenbrichter/ragmask#19]
People are making ragmasks. Some are even making further modifications to the design. Is there room for a community to be grown around it and associated with the main site?

**Comment by lorenbrichter** [comment]
Yes! I was just looking and there's a Wiki feature here on Github that folks could contribute to, and we can link to from the homepage. Maybe something like that?

**Comment by paul-hammant** [comment]
Or subdomains: remixed.ragmask.com. That could be Jekyll on GH, or wordpress, or blogger or somesuch.  

Or even a directory within ragmask.com/remixed/ 

You're hosting on Digital Ocean? What choices for dynamic or static gen'd would you consider?

**Comment by lorenbrichter** [comment]
The site is on Netlify, we could so a subfolder (but then we're dealing with pull requests every time someone wants to make a change). Let's start with a Wiki, since everyone can contribute / edit, and if it proves to be popular and the wiki is the limiting factor, we can upgrade to something more advanced.

https://github.com/lorenbrichter/ragmask/wiki
